### ISSUE TITLE
Missing LICENSE

### ISSUE THREAD

**Issue by varanauskas** [wesbos/dad-jokes#261]
I want to deliver these punchlines in usage on personal and possibly professional basis in good faith

Could you please add appropriate licensing information so people can be sure about personal and commercial usage of these jokes, their replication and execution for payment in eye-rolls and rapid exhalations through the nose?

**Comment by styfle** [comment]
@wesbos Since this repo isn't code, I suggest using something like Creative Commons.

The license.md could be:

```
[![CC0](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](https://creativecommons.org/publicdomain/zero/1.0/)

To the extent possible under law, Wes Bos has waived all copyright and related or neighboring rights to this work.
```

**Comment by SwiftyProgrammer690** [comment]
I mean, I could open a pull request!

**Comment by SwiftyProgrammer690** [comment]
Yeah, I just did it: https://github.com/wesbos/dad-jokes/pull/267

**Comment by SwiftyProgrammer690** [comment]
This issue will be closed when we have @wesbos' approval

**Comment by SwiftyProgrammer690** [comment]
Never mind, the pull request I created is very stale.
